build.gradle.kts

```diff
@@ -28,11 +34,8 @@ dependencies {
     // implementation("org.apache.lucene:lucene-analyzers-common:x")
 
     //implementation("org.slf4j:slf4j-api:1.7.6")
-    if(project.hasProperty("tests")) {
-        implementation("org.slf4j:slf4j-api:1.7.6")
-    } else {
-        implementation("de.psdev.slf4j-android-logger:slf4j-android-logger:1.0.5")
```

Comment:
When using in Android app (AndBible) we need this logger implementation.

Reply:
Could you provide PR for AndBible how to use this if we make this change?

Reply:
Well, I'm not familiar with android development so I'm not equipped to run tests on and-bible. (I could take time to try it later on, though).

But the change is very simple : you can just add the dependency directly in this list : https://github.com/AndBible/and-bible/blob/develop/app/build.gradle.kts#L335 
This should be enough :-)